src/main/scala/org/broadinstitute/dsde/firecloud/FireCloudServiceActor.scala

```diff
@@ -74,7 +82,7 @@ class FireCloudServiceActor extends HttpServiceActor with FireCloudDirectives {
         userService.routes ~
         nihSyncService.routes ~
         healthService.routes ~
-        libraryService.routes ~
+        libraryRoutes ~
```

Comment:
Why is it better to include LibraryService as a trait (and refer to the routes in this fashion) instead of using a val for libraryService?